pip/vcs/__init__.py

```diff
@@ -324,12 +324,27 @@ def run_command(self, cmd, show_stdout=True, cwd=None,
                                    command_desc, extra_environ,
                                    spinner)
         except OSError as e:
-            # errno.ENOENT = no such file or directory
-            # In other words, the VCS executable isn't available
-            if e.errno == errno.ENOENT:
-                raise BadCommand('Cannot find command %r' % self.name)
+            # on windows inform the user that self.name is not
+            # in the path environment variable.
+            if sys.platform == "win32":
+                if e.errno == errno.ENOENT:
+                    logger.error(
+                        "'{0}' is not in the path environment "
+                        "variable. Be sure to add manually or "
+                        "by using pthmgr program which is open"
+                        " source. The pre-built version can be"
+                        " found at https://github.com/Decorate"
+                        "rBot-devs/ffmpus/blob/master/pthmgr.e"
+                        "xe and the source code can be found h"
+                        "ere as well https://github.com/Decora"
+                        "terBot-devs/pthmgr.".format(self.name))
```

Comment:
Strong -1 on the advert for pthmgr. Users can manage PATH in whatever way they like - it's not pip's job to advise on this.

You're also making the assumption that the user has the VCS installed but not in PATH - surely it's equally possible that they didn't install it? A much simpler change would be to just improve the message a little (on all platforms): "Cannot find command %r - do you have it installed and on PATH?"